CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changes
 
+## 0.23.0 (2024-07-03)
+
+- [`OF#4423`][OF#4423] Added layout dropdown (single/double column) for addressNL field. [OF#4423]:
+  https://github.com/open-formulieren/open-forms/issues/4423
```

Comment:
this doesn't look good, I think you need to put the link definition on its own line ;)